flexget/plugins/cli/series.py

```diff
@@ -270,29 +292,29 @@ def register_parser_arguments():
     list_parser = subparsers.add_parser('list', parents=[table_parser],
                                         help='List a summary of the different series being tracked')
     list_parser.add_argument('configured', nargs='?', choices=['configured', 'unconfigured', 'all'],
-                             default='configured',
                              help='Limit list to series that are currently in the config or not (default: %(default)s)')
-    list_parser.add_argument('--premieres', action='store_true',
+    list_parser.add_argument('--premieres', action='store_const', const='premieres',
                              help='limit list to series which only have episode 1 (and maybe also 2) downloaded')
     list_parser.add_argument('--new', nargs='?', type=int, metavar='DAYS', const=7,
                              help='Limit list to series with a release seen in last %(const)s days. number of days can '
                                   'be overridden with %(metavar)s')
     list_parser.add_argument('--stale', nargs='?', type=int, metavar='DAYS', const=365,
                              help='Limit list to series which have not seen a release in %(const)s days. number of '
                                   'days can be overridden with %(metavar)s')
-    list_parser.add_argument('--sort-by', choices=('name', 'age'), default='name',
-                             help='Choose list sort attribute')
+    list_parser.add_argument('--sort-by', choices=('name', 'age'), help='Choose list sort attribute')
     order = list_parser.add_mutually_exclusive_group(required=False)
-    order.add_argument('--descending', dest='order', action='store_true', help='Sort in descending order')
-    order.add_argument('--ascending', dest='order', action='store_false', help='Sort in ascending order')
+    order.add_argument('--descending', dest='order', action='store_const', const='desc', help='Sort in descending order')
+    order.add_argument('--ascending', dest='order', action='store_const', const='asc', help='Sort in ascending order')
 
     show_parser = subparsers.add_parser('show', parents=[series_parser, table_parser],
                           help='Show the releases FlexGet has seen for a given series')
-    show_parser.add_argument('--sort-by', choices=('age', 'identifier'), default='age',
+    show_parser.add_argument('--sort-by', choices=('age', 'identifier'),
                              help='Choose releases list sort: age (default) or identifier')
     show_order = show_parser.add_mutually_exclusive_group(required=False)
-    show_order.add_argument('--descending', dest='order', action='store_true', help='Sort in descending order')
-    show_order.add_argument('--ascending', dest='order', action='store_false', help='Sort in ascending order')
+    show_order.add_argument('--descending', dest='order', action='store_const', const='desc',
```

Comment:
you could probably reuse these args if you really really wanted to :-)
you can set a parent parser and inherit it, but it's really a minor issue.